content/docs/glossary/versioning.mdx

```diff
@@ -26,21 +31,29 @@ A node considers a step invalid either because they do not understand it or some
 Upgrade steps are applied before applying the transaction set to ensure that the logic scheduling steps is the same that is processing it. Otherwise, the steps would have to be applied after the ledger is closed.
 
 ### Supported versions
+
 Each node has its own way of tracking which version it supports--for example, a "min version", "max version"--but it can also include things like "black listed versions." Supported versions are not tracked from within the protocol.
 
 Note that minProtocolVersion is distinct from the version an instance understands:
 typically an implementation understands versions n .. maxProtocolVersion, where n <= minProtocolVersion.
 The reason for this is that nodes must be able to replay transactions from history (down to version 'n'), yet there might be some issue/vulnerability that we don't want to be exploitable for new transactions.
 
 ### Ledger object versioning
+
 Data structures that are likely to evolve over time contain the following extension point:
-```C++
+
+<CodeExample>
+
+```cpp
```

Comment:
@rice2000, Prism recommends `cpp` for `C++`. See under "Supported languages" on https://prismjs.com/